JSON/JSON_Iterator.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * This function iterates over each property in a JSON object and logs the key and value.
+ *
+ * @example
+ * // Output:
+ * // key1: value1
+ * // key2: value2
+ * // key3: value3
+ */
+let jsonObject = {
```

Comment:
I would recommend changing this to `var`, as people might want to use it in global or other scopes that are not set to support ES2021. Thanks.

Reply:
@Lacah - Thanks for your feedback. Based on feedback, I have changed from let to var.

Reply:
Thanks!